kubeflow/example/readme.md

```diff
@@ -7,6 +7,14 @@ The main objective of this pipeline is to perform the model training for a drivi
 
 The pipeline is composed of the following components:
 
+### Connecting to MinIO
+First of all, it is necessary to connect with the [MinIO](https://minio.platform.flexigrobots-h2020.eu/) server. For this reason, the user has to introduce the identification credentials. For security reasons, these values are read from environment variables. Accordingly, an .env file has to be created inside the      [example](kubeflow/example/) folder. The credentials values in the .env are: 
```

Comment:
The "example" link is still broken since the context is missing:
![image](https://user-images.githubusercontent.com/57658959/183014241-86921e1b-37f5-4609-a120-7906db4169df.png)
Also, there are some blank spaces before its definition